src/main/java/frc/lib/team3061/drivetrain/Drivetrain.java

```diff
@@ -666,10 +666,157 @@ public boolean isMoveToPoseEnabled() {
     return this.isMoveToPoseEnabled;
   }
 
+  // method to convert swerve module number to location
+  private String getSwerveLocation(int swerveModuleNumber) {
+    switch (swerveModuleNumber) {
+      case 0:
+        return "FL";
+      case 1:
+        return "FR";
+      case 2:
+        return "BL";
+      case 3:
+        return "BR";
+      default:
+        return "UNKNOWN";
+    }
+  }
+
   private Command getSystemCheckCommand() {
-    return Commands.sequence(Commands.sequence(Commands.waitSeconds(0.25)))
-        .until(() -> !FaultReporter.getInstance().getFaults(SUBSYSTEM_NAME).isEmpty())
-        .andThen(Commands.runOnce(() -> drive(0, 0, 0, true, false), this));
+    return Commands.sequence(
+        // Tests for driving to the left
```

Comment:
The coordinate systems with which we work are a bit tricky. Refer to [this document](https://docs.google.com/document/d/17dg5cIfqVOlQTTbo2ust4QxTZlUoPNzuBu2oe58Ov84/edit#heading=h.x4ppzp81ed1) for details. From the robot frame of reference, the positive x direction is forward. Before we start driving, we should disable field relative mode so that everything is robot relative.